packages/stylelint-config/src/stylelint.test.ts

```diff
@@ -0,0 +1,81 @@
+import * as stylelint from 'stylelint';
+import griffelStylelintConfig from './index';
+import { createSyntax } from '@griffel/postcss-syntax';
+
+describe('stylelint', () => {
+  it('should lint makeStyles with standard config', async () => {
+    expect.assertions(1);
+    const config = {
+      ...griffelStylelintConfig,
+      rules: {
+        'selector-anb-no-unmatchable': [true],
+      },
+    };
+
+    const code = `
+import { makeStyles } from '@griffel/react';
+
+const useStyles = makeStyles({
+  root: {
+    ':nth-child(0)': {
+      color: 'red',
+    }
+  }
+})
+`;
+
+    const { results } = await stylelint.lint({ code, config });
+    const warnings = results.map(result => result.warnings).flat();
+    expect(warnings).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "column": 11,
+          "endColumn": 24,
+          "endLine": 5,
+          "line": 5,
```

Comment:
The starting line is always correct, however the rest of the locations are wrong because most of the stylelint default rules calculate additional offsets internally for better reporting when linting real CSS files.

This doesn't work too well with the griffel postcss syntax since we lose the mapping to the entire style slot. Example:

https://github.com/stylelint/stylelint/blob/0b686f289ab436eac64aaa78eb1646d38a153294/lib/rules/selector-anb-no-unmatchable/index.js#L91C8-L100

That rule even uses a third party parser for the selector and doesn't really use postcss at all